themes/base/accordion.css

```diff
@@ -21,3 +21,14 @@
 	border-top: 0;
 	overflow: auto;
 }
+
+/* RTL support */
+.ui-accordion-rtl {
+	direction: rtl;
+}
+.ui-accordion-rtl .ui-accordion .ui-accordion-header {
```

Comment:
This is overly specific (and actually wrong). `.ui-accordion-rtl .ui-accordion-header` is what you actually want.

I'd personally prefer just changing the rule to `padding: .5em` and not having anything for RTL here.